### PR TITLE
Added the tutorial for NFT deployment and token Mint

### PR DESCRIPTION
**Pull request by aditya-passbird**
https://metakeep-dev.readme.io/reference/create-your-first-nft-collection-and-mint-tokens

### CODE REVIEW

File: lambda/voting/.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-API_KEY = 
+API_KEY = // Your API key from https://console.metakeep.xyz
```
Comment (rohan-passbird) [inline lambda/voting/.env:1]:
```suggestion
// Your MetaKeep API key
API_KEY = 
```

File: lambda/voting/scripts/create.mjs [hunk 2]
```diff
@@ -7,7 +7,7 @@ import getDeveloperWallet, {
   getTransactionStatus,
   sleep,
   checkAPIKey,
-} from "./utils.mjs";
+} from "../../../helpers/utils.mjs";
```
Comment (rohan-passbird) [inline lambda/voting/scripts/create.mjs:10]: update transaction check logic in this file to use the utils

File: nfts/.env [hunk 3]
```diff
@@ -0,0 +1,6 @@
+API_KEY = // Your API key
```
Comment (rohan-passbird) [inline nfts/.env:1]:
```suggestion
// Your MetaKeep API key
API_KEY =
```

File: nfts/scripts/deployAndMintNFT.js [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  checkAPIKey();
+
+  /* *************************************************************** Create Collection *************************************************************** */
+  console.log(
+    "***************************************************** Create Collection *****************************************************\n"
+  );
+  // Creates a New Collection
+  console.log("Creating a new collection...");
+  const resultJson = await createCollection();
+  console.log("Collection created successfully!");
```
Comment (rohan-passbird) [inline nfts/scripts/deployAndMintNFT.js:131]:
```suggestion
  console.log("Collection creation in progress...");
```

File: nfts/scripts/deployAndMintNFT.js [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+  // Waits for the transaction to be mined
+  await waitUntilTransactionMined(resultJson);
+  console.log(
+    "NFT Collection created successfully! Collection Address: " +
+      resultJson.collection +
+      "\n"
+  );
+
+  // Waits for 5 seconds.
+  sleep(5000);
```
Comment (rohan-passbird) [inline nfts/scripts/deployAndMintNFT.js:142]: Why are we adding a 5 sec gap between create and mint?

File: nfts/scripts/deployAndMintNFT.js [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+  console.log(
+    "***************************************************** Mint NFT *****************************************************\n"
+  );
+
+  // Mints a new NFT
+  console.log("Minting a new NFT...");
+
+  const collection = resultJson.collection;
+
+  const resultJson2 = await mintNFT(collection);
```
Comment (rohan-passbird) [inline nfts/scripts/deployAndMintNFT.js:154]:
```suggestion
  const mintResultJson = await mintNFT(collection);
```

File: nfts/scripts/deployAndMintNFT.js [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+  console.log("NFT minted successfully!");
```
Comment (rohan-passbird) [inline nfts/scripts/deployAndMintNFT.js:156]:
```suggestion
  console.log("NFT mint in progress....");
```

File: nfts/README.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+git clone https://github.com/PassbirdCo/MetaKeepCodeSamples.git
+cd nfts
+npm install
+```
+
+Then, on a new terminal, go to the repository's `nft` folder and run this to
+deploy NFT contract and mint token:
+
+```sh
+npm run deployAndMint
```
Comment (rohan-passbird) [inline nfts/README.md:21]:
```suggestion
npm run createCollectionAndMintToken
```

File: nfts/scripts/deployAndMintNFT.js [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+  console.log(
+    "NFT minted successfully! Transaction Hash: " +
+      resultJson.transactionHash +
+      "\n" +
+      "NFT Token Id:" +
+      resultJson2.token +
+      "\n"
+  );
+}
+
```
Comment (rohan-passbird) [inline nfts/scripts/deployAndMintNFT.js:171]: also add opensea link here

File: nfts/scripts/deployAndMintNFT.js (hunk 5 above)
Comment (aditya-passbird) [reply nfts/scripts/deployAndMintNFT.js:142]: just to give small pause between the create and mint.

File: nfts/scripts/deployAndMintNFT.js (hunk 9 above)
Comment (aditya-passbird) [reply nfts/scripts/deployAndMintNFT.js:176]: done

File: lambda/voting/scripts/create.mjs (hunk 2 above)
Comment (aditya-passbird) [reply lambda/voting/scripts/create.mjs:9]: done

File: nfts/package.json [hunk 10]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nfts",
+  "version": "1.0.0",
+  "description": "End to End tutorials for NFT deployment and minting",
+  "main": "scripts/index.js",
+  "scripts": {
+    "deployAndMint": "node scripts/deployAndMintNFT.js",
```
Comment (rohan-passbird) [inline nfts/package.json:7]: I belive need to update this . also rename the file

File: nfts/scripts/deployAndMintNFT.js [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+      "/" +
+      mintResultJson.token +
+      "\n" +
+      "OpenSea Link (PRODUCTION): https://opensea.io/assets/matic/" +
+      collection +
+      "/" +
+      mintResultJson.token +
+      "\n"
+  );
+}
```
Comment (rohan-passbird) [inline nfts/scripts/deployAndMintNFT.js:180]: Why is this one big console.log?
Split this in multiple lines. `\n` will be automatically added

Comment (aditya-passbird) [reply nfts/scripts/deployAndMintNFT.js]: done

File: nfts/package.json (hunk 10 above)
Comment (aditya-passbird) [reply nfts/package.json]: done

**Review by rohan-passbird — APPROVED** [review]
👍

**Pull request merged**